sdk/trace/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorTest.java

```diff
@@ -317,52 +329,47 @@ void exporterThrowsException() {
 
   @Test
   @Timeout(5)
-  public void exporterTimesOut() throws InterruptedException {
```

Comment:
I don't think this test was actually checking what it wanted to check, or at least I couldn't reason about it well. Tried rewriting it.